pkg/transforms/policy.go

```diff
@@ -11,7 +11,11 @@ Copyright (c) 2020 Red Hat, Inc.
 package transforms
 
 import (
+	"strings"
+
 	p "github.com/stolostron/governance-policy-propagator/api/v1"
```

Comment:
(Optional) This is too short in my opinion--it'd be nice to have `policyv1` (or just `policy`).